packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -284,7 +284,7 @@ async function askAdditionalQuestions(context, authConfig, defaultAuthType, mode
 async function askResolverConflictQuestion(context, modelTypes?) {
   let resolverConfig: any = {};
 
-  if (await context.prompt.confirm('Configure conflict detection?')) {
+  if (await context.prompt.confirm('Configure conflict detection ? (required for Amplify DataStore)')) {
```

Comment:
remove the whitespace before the "?" And call it only "DataStore".

Reply:
```suggestion
  if (await context.prompt.confirm('Configure conflict detection? (required for DataStore)')) {
```